lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -167,6 +173,7 @@ def create_setup_intent(payment_method, options = {})
               return result if result.is_a?(ActiveMerchant::Billing::Response)
             end
 
+            add_network_token_info(post, payment_method, options)
```

Comment:
Before we were sending cryptogram and ECI for AP/GP through add_digital_wallet as seen [here ](https://github.com/activemerchant/active_merchant/blob/d6b450082775dcead08203fc10e50bc2e605d75a/lib/active_merchant/billing/gateways/stripe_payment_intents.rb#L451) but that was removed in [this change](https://github.com/activemerchant/active_merchant/commit/3d28e306764e38be97cc7a552f46aa5070039550#diff-19492101b546175aa3ca6a16ec7d0a610d0adae03cbaf130eff54092dec8e0fbL451) so I'm fixing that in this PR